LLVM.Imports.Disassembler.pas

```diff
@@ -2,15 +2,31 @@
 
 interface
 
-//based on Disassembler.h
+//based on Disassembler.h and  DisassemblerTypes.h
```

Comment:
any particular reason you put both files into one pas? I'd prefer to have it split to match the file layout of the c headers

Reply:
no no reason, I wanted to leave only one file as originally for old versions. anyway I modified